dev-ruby/certificate_authority/certificate_authority-1.0.0.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2021 Gentoo Authors
```

Comment:
These kinds of changes (EAPI, adding a patch) require a new revision of the ebuild.